modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastFragment.kt

```diff
@@ -871,6 +873,15 @@ class PodcastFragment : BaseFragment() {
         return binding.root
     }
 
+    private fun showPodrollInformationModal() {
+        val dialog = ConfirmationDialog()
+            .setIconId(R.drawable.ic_author)
+            .setTitle(getString(LR.string.podroll_information_title))
+            .setSummary(getString(LR.string.podroll_information_summary))
+            .setButtonType(ConfirmationDialog.ButtonType.Normal(getString(LR.string.podroll_information_button)))
+        dialog.show(parentFragmentManager, "podroll_information")
```

Comment:
When I quickly double tap the header, the dialog shows up twice. It would be nice to check if a dialog like this doesn't already show in the `FragmentManager`.

Reply:
I have overridden the show method in the confirmation dialog with the following. I'm slightly concerned as it is used in lots of places. Does this look ok to you? 
```
        if (manager.findFragmentByTag(tag) == null) {
            super.show(manager, tag)
        }
```